test/src/main/java/org/apache/accumulo/test/functional/FateStarvationIT.java

```diff
@@ -83,20 +85,26 @@ public void run() throws Exception {
 
       List<Text> splits = new ArrayList<>(TestIngest.getSplitPoints(0, 100000, 67));
 
-      List<Future<?>> futures = new ArrayList<>();
+      int numTasks = 100;
+      List<Future<?>> futures = new ArrayList<>(numTasks);
       var executor = Executors.newCachedThreadPool();
+      CountDownLatch startLatch = new CountDownLatch(32); // wait for a portion of the tasks to be
+                                                          // ready
 
-      for (int i = 0; i < 100; i++) {
+      for (int i = 0; i < numTasks; i++) {
         int idx1 = RANDOM.get().nextInt(splits.size() - 1);
         int idx2 = RANDOM.get().nextInt(splits.size() - (idx1 + 1)) + idx1 + 1;
 
         var future = executor.submit(() -> {
-          c.tableOperations().compact(tableName, splits.get(idx1), splits.get(idx2), false, true);
+          startLatch.countDown();
+          startLatch.await();
+          c.tableOperations().compact(tableName, splits.get(idx1), splits.get(idx2), false, false);
```

Comment:
```suggestion
          c.tableOperations().compact(tableName, splits.get(idx1), splits.get(idx2), false, true);
```
We still want to wait for these compactions to complete, otherwise the completion of the future doesn't necessarily mean the completion of the compaction, which the test relies on